docs/en/operations/table_engines/versionedcollapsingmergetree.md

```diff
@@ -0,0 +1,226 @@
+<a name="table_engine-versionedcollapsingmergetree"></a>
+
+# VersionedCollapsingMergeTree
```

Comment:
Cannot figure out the motivation for `VersionedCollapsingMergeTree` from this text.